Mage/src/main/java/mage/abilities/hint/common/BoastHint.java

```diff
@@ -0,0 +1,28 @@
+package mage.abilities.hint.common;
+
+import mage.abilities.Ability;
+import mage.abilities.condition.common.BoastCondition;
+import mage.abilities.hint.ConditionHint;
+import mage.abilities.hint.Hint;
+import mage.game.Game;
+
+/**
+ *
+ * @author weirddan455
+ */
+public enum BoastHint implements Hint {
+    instance;
+    private static final ConditionHint hint = new ConditionHint(BoastCondition.instance,
+            "Can activate Boast ability", null,
+            "Can't activate Boast ability", null, true);
```

Comment:
If you use conditional hint then no needs to make `can/can't` texts -- just use can text with icons (by default). Fixed in 6b3de78db889e09cb28bfe80ec0458ab4fb7c69c